### PR TITLE
Update when to predict another action

### PR DESCRIPTION
**Pull request by ricwo**
**Proposed changes**:
- Do not predict another action when latest run action was action_session_start

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: tests/core/test_actions.py [hunk 1]
```diff
@@ -553,6 +550,8 @@ def test_domain_action_instantiation():
     # make sure that the list of events has ascending timestamps
     assert sorted(events, key=lambda x: x.timestamp) == events
```
Comment (ricwo) [inline tests/core/test_actions.py]:
```suggestion
```

File: rasa/core/processor.py [hunk 2]
```diff
@@ -142,6 +142,33 @@ def __init__(
...
+
+        Args:
+            tracker: Tracker to run the actions on.
+            output_channel: Output channel for potential utterances in a custom
+                `ActionSessionStart`.
+        """
+
+        action = self._get_action(ACTION_SESSION_START_NAME)
+
+        while action:
```
Comment (wochinge) [inline rasa/core/processor.py]: I feel it's weird if we haven another (dummy) prediction loop in the code. Imo the condition for `should_predict_another_action` is wrong and should be `should_predict_another_action(tracker.latest_action_name)`. What do you think?

Comment (ricwo) [reply rasa/core/processor.py]: agreed - updated 👍

File: rasa/core/processor.py [hunk 3]
```diff
@@ -480,9 +480,19 @@ def is_action_limit_reached():
 
     # noinspection PyUnusedLocal
     @staticmethod
-    def should_predict_another_action(action_name, events) -> bool:
-        is_listen_action = action_name == ACTION_LISTEN_NAME
-        return not is_listen_action
+    def should_predict_another_action(action_name: Text, events: List[Event]) -> bool:
```
Comment (wochinge) [inline rasa/core/processor.py]: how about removing the `events` parameter since it's not used?

File: rasa/core/processor.py [hunk 4]
```diff
@@ -480,9 +480,19 @@ def is_action_limit_reached():
...
+        Args:
+            action_name: Name of the latest executed action.
+            events: List of events returned by the latest executed action.
+
+        Returns:
+            `False` if `action_name` is `ACTION_LISTEN_NAME` or
+            `ACTION_SESSION_START_NAME`, otherwise `True`.
+        """
+
+        return action_name not in (ACTION_LISTEN_NAME, ACTION_SESSION_START_NAME)
```
Comment (wochinge) [inline rasa/core/processor.py]: optional: alternatively, we could also pass `tracker.last_action ` as `action_name` (then you can keep the condition here as before).

**Pull request merged**